packages/contracts/scripts/deploy/ubiquistick/TheUbiquityStick.ts

```diff
@@ -0,0 +1,25 @@
+import { DeployFuncParam } from "../../shared";
+import { create } from "../create"
+
+export const optionDefinitions = [
+    { name: 'task', defaultOption: true },
+    { name: "network", alias: 'n', type: String },
+]
+
+const func = async (params: DeployFuncParam) => {
+    const contractInstance = "src/ubiquistick/TheUbiquityStick.sol:TheUbiquityStick";
+    const { env, args } = params;
+    const { result, stderr } = await create({ ...env, name: args.task, network: args.network, contractInstance, constructorArguments: [] });
+
+    // TODO: Do we need to set tokenURI during the deployment? For example, if we should have 10k tokens, 
+    // this part will definitely be an issue to consume lots of gas. General idea is to set baseURI and others are getting generated
+    // from baseURI automatically. So it might be a way to have them as a forge script.
+    //
+    // prev source code:  
+    // 
+    // await theUbiquityStick.connect(deployer).setTokenURI(0, tokenURIs.standardJson);
+    // await theUbiquityStick.connect(deployer).setTokenURI(1, tokenURIs.goldJson);
+    // await theUbiquityStick.connect(deployer).setTokenURI(2, tokenURIs.invisibleJson);
+    return !stderr ? "succeeded" : "failed"
```

Comment:
I see a lot of repetitive logic, I would consider making a utility function to handle every instance. For example, I imagine you could generate the `contractInstance` value by passing in `__filename` into this utility function. To clarify, this isn't a blocker; we can definitely merge without clean deploy scripts but its just something that I personally would have done.

Reply:
I also tried to avoid repetition in codebase but what I was considering here is that there might be some cases we have to parse them differently.  yeah you can definitely create a new issue with clear instruction.